CDCgov/prime-reportstream
# Remove frontend dummy Github Action #12288

This PR removes the "dummy" action from the Build Frontend workflow. 

Test Steps:
1. Open up a PR with changes to one or more of the following paths:
            - 'frontend-react/**'
            - '.github/actions/build-vars/action.yml'
            - '.github/actions/build-frontend/action.yml'   
   Build Frontend should run as part of the checks and actually build the FE.
2. Open up a PR *without* any changes to the paths above. Build Frontend should **still** run, but quicker: should skip building the FE.

## Changes
- Removed `build_frontend_dummy.yaml`. Used conditionals to either build or skip frontend within build_frontend.yaml.

## Checklist

### Testing
## Linked Issues
- Fixes #12288
